docs/multidatacenter.md

```diff
@@ -142,6 +142,18 @@ Then the cassandra node distribution would probably end up similar to this:
 - 4x `us-west-2b`
 - 4x `us-west-2c`
 
+## Adding instances running in other Kubernetes clusters
+
+Datacenters can also span multiple Kubernetes clusters. To let an instance know about a datacenter running in another Kubernetes cluster, the `EXTERNAL_SEED_NODES` parameter has to be set. This parameter takes an array of DNS names or IP addresses that seed nodes outside of the cluster have. The clusters have to be set up so that the pods running Cassandra nodes can communicate with each other.
```

Comment:
This needs a hint that the CLUSTER_NAME needs to be the same (I think)

Reply:
Good point! I didn't check that yet, because I was using the same cluster name in all my tests. Will add this here.